src/client/components/mapComponents/Map.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import MapBase from './MapBase';
+import Marker from './Marker';
+import Icons from './Icons';
+import PropTypes from 'prop-types';
+
+const Map = ({ center, markers, currentPosition, style }) => {
```

Comment:
It shows a warning since 'style' is not used. It could be removed if its not required.